src/sales/reorders-allow.md

```diff
@@ -32,3 +32,61 @@ _Customer reorder link in the Admin_
    No|Disables reorders.
 
 1. Click <span class="btn">Save Config</span>.
+
+## Reorder from Storefront
+
+Customer may initiate the reorder functionality for a particular order from the
+
+- “My Orders” page
+
+- “Order View” page
+
+## My Orders
+
+“Reorder“ button is always displayed in the list with Orders (even if all products from the order is not available for reorder).
+
+![]({% link images/images/My order page view.png %}){: .zoom}
+
+**Case 1.** All products from the order are **available** for Reorder
+
+User will be redirected to the Cart and all products are added to the Cart
+
+![]({% link images/images/shopping-cart-page.png %}){: .zoom}
+
+**Case 2.** Some/all products from the order are **non-available** for Reorder
+
+The "Reorder" button will not appear on the "My Orders" and "View Order" pages.
+
+![]({% link images/images/my-orders-view-page1.png %}){: .zoom} 
```

Comment:
@EvheniiaKolkhiieva , please remove a space at the end of sentence.

Thank you!